app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError

```

Comment:
There is no need to duplicate `import` statements three times to import three entities from one module. Import all required errors at once; you could place them inside of `()` and place each error on a new line to improve the readability.